### ISSUE TITLE
Stop the webpack watcher when the electron app is closed

### ISSUE THREAD

**Issue by osechet** [colinskow/angular-electron-dream-starter#4]
* **I'm submitting a ...**
[ ] bug report
[x] feature request
[ ] question about the decisions made in the repository

* **What is the current behavior?**
When closing the electron app, the webpack watcher remains running, forcing the dev to interrupt it.

* **What is the expected behavior?**
It would be great if the webpack watcher could stop at the same time as the electron app.

* **What is the motivation / use case for changing the behavior?**
Make life of the dev easier ;)

* **Please tell us about your environment:**
angular-electron-dream-starter's master on Ubuntu 14.04

**Comment by colinskow** [comment]
I agree, please let me know if you can figure out how to do this! It may require a PR to `electron-connect`.

**Comment by Kaffiend** [comment]
I've been seeing this a lot too. When you try to restart rimraf has a fit when one of the directories is in use.
```
Error: EPERM: operation not permitted, unlink 'B:\appl\ics\ian\nge_client\dev\node_modules\symbol-observable'
    at Error (native)
```
I've been reloading the editor to clear up the binding.

**Comment by Kaffiend** [comment]
disregard, different issue, it is the angular language service causing the rimraf on restart to complain.

**Comment by Kaffiend** [comment]
this could possibly be handled by adding `nodemon`  to wrap the process, and can also be configured to restart on just electron main process changes.
